string msg
string command
---
string msg